### PR TITLE
fea(plug): add general tensorflow function

### PR DESCRIPTION
**Pull request by superrxan**
Signed-off-by: Jianxiang Ran <rxan_embedded@163.com>

### CODE REVIEW

File: extensions/functions/tensorFlowGeneral/resize.go [hunk 1]
```diff
@@ -0,0 +1,79 @@
+// Copyright 2022 EMQ Technologies Co., Ltd.
```
Comment (ngjaying) [inline extensions/functions/tensorFlowGeneral/resize.go:1]: Try to merge this to the original resize function or remove from this PR. Let the resize function to have args of (x,y,z, channels, format)

File: extensions/functions/tensorFlowGeneral/install.sh [hunk 2]
```diff
@@ -0,0 +1,40 @@
+#!/bin/sh
```
Comment (ngjaying) [inline extensions/functions/tensorFlowGeneral/install.sh:1]: In the etc tflite example models, can we add a license file from the original location?

File: extensions/functions/tensorFlowGeneral/tflite.go [hunk 3]
```diff
@@ -0,0 +1,259 @@
...
+				}
+				err = input.SetUint8s(v)
+				if err != nil {
+					return nil, false
+				}
+			default:
+				return fmt.Errorf("invalid %d parameter, unsupported type %v in the model", i, t), false
+			}
+		default:
+			// TODO support multiple dimensions. Here assume user passes a 1D array.
```
Comment (ngjaying) [inline extensions/functions/tensorFlowGeneral/tflite.go:156]: Here add a validation for the input argument like the first case

**Review by ngjaying — CHANGES_REQUESTED** [review]

File: extensions/functions/tensorFlowGeneral/resize.go (hunk 1 above)
Comment (superrxan) [reply extensions/functions/tensorFlowGeneral/resize.go:1]: ack

File: extensions/functions/tensorFlowGeneral/tflite.go (hunk 3 above)
Comment (ngjaying) [reply extensions/functions/tensorFlowGeneral/tflite.go:156]: wrong validation, len(arg) == dim(1) * dim(2) * dim(n)

**Review by ngjaying — CHANGES_REQUESTED** [review]

**Review by ngjaying — APPROVED** [review]

**Pull request merged**